### PR TITLE
Fix indices

### PR DESCRIPTION
**Pull request by jayrbolton**

### CODE REVIEW

File: spec/config.yaml [hunk 1]
```diff
@@ -494,12 +494,12 @@ mappings:
       starts: {type: integer}
       strands: {type: keyword}
       stops: {type: integer}
-      functions: {type: text}
-      functional_descriptions: {type: text}
-      warnings: {type: text}
+      functions: {type: keyword}
```
Comment (slebras) [inline spec/config.yaml:497]: Are these being changed back to what they were. I changed them earlier to match up with the previous versions of the indexers

File: src/index_runner/es_indexers/annotated_metagenome_assembly.py [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
-_AMA_FEATURES_INDEX_VERSION = 1
+_AMA_INDEX_VERSION = 2
+_AMA_FEATURES_INDEX_VERSION = 2
 _AMA_INDEX_NAME = "annotated_metagenome_assembly_" + str(_AMA_INDEX_VERSION)
 _AMA_FEATURES_INDEX_NAME = "annotated_metagenome_assembly_features_" + str(_AMA_FEATURES_INDEX_VERSION)
 # version indices
-_VER_AMA_INDEX_VERSION = 1
-_VER_AMA_FEATURES_INDEX_VERSION = 1
+_VER_AMA_INDEX_VERSION = 2
+_VER_AMA_FEATURES_INDEX_VERSION = 2
```
Comment (slebras) [inline src/index_runner/es_indexers/annotated_metagenome_assembly.py:19]: are there other changes to this indexer

File: src/index_runner/es_indexers/genome.py [hunk 3]
```diff
@@ -3,8 +3,8 @@
 from src.utils.config import config
 
 _NAMESPACE = "WS"
-_GENOME_INDEX_VERSION = 1
-_GENOME_FEATURE_INDEX_VERSION = 2
+_GENOME_INDEX_VERSION = 2
+_GENOME_FEATURE_INDEX_VERSION = 3
```
Comment (slebras) [inline src/index_runner/es_indexers/genome.py:7]: are there other changes to this indexer

File: src/index_runner/es_indexers/narrative.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 _NAMESPACE = "WS"
 _MARKDOWNER = Markdown()
-_NARRATIVE_INDEX_VERSION = 1
+_NARRATIVE_INDEX_VERSION = 2
```
Comment (slebras) [inline src/index_runner/es_indexers/narrative.py:15]: are there other changes to this indexer

File: spec/config.yaml (hunk 1 above)
Comment (jayrbolton) [reply spec/config.yaml:497]: They are being changed to the type mappings we have deployed on CI. Setting all this to type:text produces a bunch of type mapping conflict errors. 

It seems like at some point things got a little confused. It seems to me like the types in this PR are the original type mappings deployed to CI and prod

File: src/index_runner/es_indexers/annotated_metagenome_assembly.py (hunk 2 above)
Comment (jayrbolton) [reply src/index_runner/es_indexers/annotated_metagenome_assembly.py:19]: Yeah, first there was: https://github.com/kbase/index_runner_spec/pull/14/files
Then I added https://github.com/kbase/index_runner_spec/pull/15/files

**Pull request merged**